### DISCUSSION TITLE
Apple TV

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#76] [General]
# Apple TV

Check if Apple TV is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/b77fc13f-8d3e-4891-b15e-0cbe5852d776

**Comment by talynone** [comment]
Sideloading manually works.

**Reply by OpenSource03** [reply]
We will mark it as native after it's officially supported via the windows store.

**Reply by talynone** [reply]
Apple TV is now an ARM64EC app in the app store. Verified on my computer.

**Comment by Guido-HH** [comment]
It works from the microsoft store

**Reply by richardseberino** [reply]
it works for me as well, I installed it from Microsoft Store. It is still x86, emulated, but it works
